src/components/atoms/Dropdown/Dropdown.tsx

```diff
@@ -54,11 +54,71 @@ interface DropdownProps {
   className?: string;
   placement?: MenuPlacement;
   noArrow?: boolean;
+  onSelect?: (option: Option) => void;
 }
 
-export const Dropdown: React.FC<DropdownProps> = ({ title, children }) => {
-  const value = remap(title);
+export type Option = {
+  label: ReactNode;
+  value: string;
+};
+
+export const Dropdown: React.FC<DropdownProps> = ({
+  title,
+  children,
+  onSelect,
+}) => {
+  const [isOpened, setOpened] = useState<boolean>(false);
+  const [selectedOption, setSelectedOption] = useState<Option>();
+
   const options = React.Children.map(children, remap) ?? ALWAYS_EMPTY_ARRAY;
 
-  return <Select value={value} placeholder={value} options={options} />;
+  const selectOption = (option: Option) => {
+    setSelectedOption(option);
+    setOpened(false);
+    onSelect?.(option);
+  };
+
+  return (
+    <>
+      <button
+        onClick={() => setOpened((value) => !value)}
+        className="text-white bg-blue-700 hover:bg-blue-800 focus:ring-4 focus:ring-blue-300 font-medium rounded-lg text-sm px-4 py-2.5 text-center inline-flex items-center dark:bg-blue-600 dark:hover:bg-blue-700 dark:focus:ring-blue-800"
+        type="button"
+      >
+        {selectedOption?.label ?? title}
+        <svg
+          className="ml-2 w-4 h-4"
+          fill="none"
+          stroke="currentColor"
+          viewBox="0 0 24 24"
+          xmlns="http://www.w3.org/2000/svg"
+        >
+          <path
+            strokeLinecap="round"
+            strokeLinejoin="round"
+            strokeWidth="2"
+            d="M19 9l-7 7-7-7"
+          />
+        </svg>
```

Comment:
Probably OOS, but have to ask if there are pros/cons to in-lining SVGs because they use Tailwind classes, questions like if it might be TW will remove those classes if the SVGs are moved outside in separate assets files